### ISSUE TITLE
7.2.4 fails to send password to Redshift when you don't have "Save password locally" checked.

### ISSUE THREAD

**Issue by danbrakeley** [dbeaver/dbeaver#10339]
#### System information: 
- Operating system: Windows 10 Pro 2004 (19041.63)
- DBeaver: v7.2.4
- Additional extensions: I don't remember ever adding any extentions.

#### Connection specification:
- Database name and version: AWS Redshift
- Driver name: Redshift
- Do you use tunnels or proxies (SSH, SOCKS, etc)? I use an SSH Tunnel

#### Describe the problem you're observing:
I have multiple saved connections that were originally created many versions ago, and immediately after upgrading to 7.2.4, I started seeing JDBC errors from the Redshift driver about how it wasn't being given a PWD.

My dev connections continued to work fine, but my prod connections all gave that error about requiring a PWD. Both sets of connections use SSH Tunneling, but only the dev connections have "Save password locally" checked. The prod connections require me to enter the password each time I connect. I believe that is an important step in the repro.

#### Steps to reproduce, if exist:
1. Create a connection using the Redshift driver, via an SSH tunnel, and uncheck "Save password local".
2. Attempt to connect with this connection. When it asks for the password, enter the appropriate password.

Result: an error complaining about PWD being required.
Expected: a connection is established.

#### Include any warning/errors/backtraces from the logs

```
[Amazon][JDBC](10100) Connection Refused: [Amazon][JDBC](11640) Required Connection Key(s): PWD; [Amazon][JDBC](11480) Optional Connection Key(s): AccessKeyID, AuthMech, AutoCreate, BlockingRowsMode, ClusterID, DbGroups, DisableIsValidQuery, DriverLogLevel, EndpointUrl, FilterLevel, ForceLowercase, IAMDuration, Language, OpenSourceSubProtocolOverride, plugin_name, profile, Region, SecretAccessKey, selectorProvider, selectorProviderArg, SessionToken, socketTimeout, sslcert, sslfactory, sslkey, sslpassword, sslrootcert, SSLTruststore , SSLTrustStorePath, tcpKeepAlive, TCPKeepAliveMinutes, unknownLength
```

**Comment by danbrakeley** [comment]
Note that uninstalling 7.2.4 and reinstalling 7.2.3 fixed the issue for me.

**Comment by uslss** [comment]
probably it is fixed in 7.2.5. Could you check if it fixed your issue?
thanks in advance

**Comment by danbrakeley** [comment]
It does appear to be fixed in 7.2.5. Thanks!
